src/QirRuntime/samples/StandaloneInputReference/qsharp/qir-standalone-input-reference.qs

```diff
@@ -1,15 +1,28 @@
 ﻿namespace Quantum.StandaloneSupportedInputs {
 
+    open Microsoft.Quantum.Arrays;
     open Microsoft.Quantum.Canon;
     open Microsoft.Quantum.Intrinsic;
 
+    function TautologyPredicate<'T> (input : 'T) : Bool
+    {
+        return true;
+    }
+
     @EntryPoint()
-    operation ExerciseInputs (intValue : Int, doubleValue : Double, resultValue : Result, stringValue : String) : Int {
-        Message("Exercise Supported Inputs");
+    operation ExerciseInputs (intValue : Int, intArray : Int[], doubleValue : Double, doubleArray : Double[], boolValue : Bool, boolArray : Bool[], pauliValue : Pauli, pauliArray : Pauli[], rangeValue : Range, resultValue : Result, resultArray : Result[], stringValue : String) : Unit {
```

Comment:
Is there a reason that string arrays and range arrays are not included in this entry point?

Reply:
When simulating an entry-point with a `String` or `Range` array, the simulation crashes. This seems to be related to the generated QIR. I'll investigate this as a separate issue and update this reference